src/main/scala/com/lightbend/paradox/apidoc/ApidocDirective.scala

```diff
@@ -94,10 +99,10 @@ class ApidocDirective(allClassesAndObjects: IndexedSeq[String], properties: Map[
         DirectiveNode.Format.Inline,
         doctype + "doc",
         label,
-        syntheticSource,
+        new DirectiveNode.Source.Direct(fqcn),
         node.attributes,
-        fqcn,
-        new TextNode(label)
+        label, // contents
+        null
```

Comment:
hrm, pretty hard to understand, but the tests speak for themselves I guess...

Reply:
It ties into the changes in Paradox, which now renders the `contents` string. API docs directives never contain nodes.

Reply:
As it turned out, I had to change apidoc to not pre-encode the label when Java-generics should show. That encoding is now applied by the `CodeNode` which I added in Paradox.